optimum/commands/export/openvino.py

```diff
@@ -343,49 +343,52 @@ def run(self):
                 )
         elif self.args.weight_format in {"fp16", "fp32"}:
             ov_config = OVConfig(dtype=self.args.weight_format)
-        elif self.args.weight_format is not None:
-            # For int4 quantization if no parameter is provided, then use the default config if exists
-            if no_compression_parameter_provided(self.args) and self.args.weight_format == "int4":
-                quantization_config = get_default_int4_config(self.args.model)
+        else:
+            if not is_nncf_available():
+                raise ImportError("Applying quantization requires nncf, please install it with `pip install nncf`")
```

Comment:
(not very important) we could also use `NNCF_IMPORT_ERROR` here https://github.com/huggingface/optimum-intel/blob/6407c9c9115e39b28f083e61d179ddd873970cd5/optimum/intel/utils/import_utils.py#L479